Makefile

```diff
@@ -30,10 +42,16 @@ helm-lint-%:
 	$(HELM) lint $(CHART_DIR)/$*
 
 deploy-cert-manager:
-	$(HELM) upgrade -i --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
+	$(HELM_RA) repo add jetstack https://charts.jetstack.io
+	docker cp $(DOCKER_NAME):/tmp/.config/helm/repositories.yaml .
+	docker cp $(DOCKER_NAME):/tmp/.cache/helm/repository/jetstack-index.yaml .
+	docker rm $(DOCKER_NAME)
+	$(HELM_CM) upgrade -i --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
```

Comment:
Have you considered combining this into a single command, something like `docker run ... sh -c "helm repo add ... && helm upgrade ...`? Then you could skip the copying of files in and out of the container.

Reply:
That is definitely a lot cleaner. Made the change.